CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -25,6 +26,24 @@ def layer_test() -> str:
     return "hello world"
 
 
+class Field_Validator:
```

Comment:
Can we just use a [named tuple](https://stackoverflow.com/q/2970608) here instead of a class? We wouldn't need to create a new class object each time, plus the class object will have a higher size than the tuple

Reply:
I'm looking here: https://www.geeksforgeeks.org/difference-between-dataclass-vs-namedtuple-vs-object-in-python/

Looks like a class object is a bit smaller in size. But with the functions I added it's probably a bit larger. In terms of read property it seems like class objects are a bit faster.

But this is just one sample

Reply:
Hmm, after reading more about this, using `DataClass` and accessing it's attributes directly rather than using getters should be the fastest way.

Reply:
@abhinav1912 could you point me to your readings? I haven't used data classes or named tuples before so I was doing this research for the first time. 

I wonder what the speed difference of A `DataClass` and an `object class` is if both accessed the attributes directly

Reply:
Just read this article: https://medium.com/@jacktator/dataclass-vs-namedtuple-vs-object-for-performance-optimization-in-python-691e234253b9

It makes sense to use DataClass for this class because there is no logic besides getting/setting

Reply:
That was the article which I was referring to, so I think we're all on the same page for this one.

Reply:
@abhinav1912 done 4909a4d969cd39b65cf003f898c624305c210b76